### DISCUSSION TITLE
Auto-update after create smth

### DISCUSSION THREAD

**Discussion by mxpnk** [dbeaver/dbeaver#10760] [Q&A] [answered]
#### System information: 
- Operating system (distribution) and version: win10
- DBeaver version: 7.3.0

#### Connection specification:
- Database name and version: postgresql 12.5

#### Describe your question:
Hello!

How to enable auto-update after creating tables or functions using commands? Reconnection doesn't work for me too. I must close and open connection every time after every create. My settings are default.

In this screenshot you can see the error that the function (and table for function too) has already been created.

![request1](https://user-images.githubusercontent.com/67520988/102869482-ba602b00-444c-11eb-8713-5863e91e0b26.png)

After disconnect and connect:

![request2](https://user-images.githubusercontent.com/67520988/102870526-31e28a00-444e-11eb-8a58-bae9b6560cde.png)

Thank you in advance. Sorry if my question is a duplicate.

**Comment by kseniiaguzeeva** [comment, marked as answer]
Now you should refresh the DB Navigator tree objects to see changes. It is a default DBeaver behaviour now and feature request exists to change it https://github.com/dbeaver/dbeaver/issues/9283.

**Reply by mxpnk** [reply]
Do you mean press 'F5' or refresh button? It doesn't work for me too, same as reconnect button. Only disconnect and connect again.

**Comment by kseniiaguzeeva** [comment]
Do you refresh functions and tables folder?

**Reply by mxpnk** [reply]
Yes, sure

**Comment by mxpnk** [comment]
After reinstalling updates by 'F5' correctly

**Comment by kseniiaguzeeva** [comment]
Nice to hear!
